### ISSUE TITLE
jasmine tests run in random order

### ISSUE THREAD

**Issue by AndreyYevtukhov** [webdriverio/wdio-jasmine-framework#124]
Hi!

After update to version 0.3.3 our tests have started to fail unexpectedly.
We have checked why, and found out, that tests were running in a different order each time.

Reverting to 0.3.2 fixed this issue/feature.
(maybe this issue will help someone to figure it out faster)

---

If there was any flag added to the configuration, or something changed - please update plugin description docs.

---
If you need more information about the issue - I will try to give it here.

**Comment by Mechanicalkeyboard** [comment]
Seconded

**Comment by christian-bromann** [comment]
@Mechanicalkeyboard this change was introduced in Jasmine, not WebdriverIO. In fact WebdriverIO is not responsible for executing the specs in any order.

**Comment by dudaming** [comment]
Hello @christian-bromann 
It is from Jasmine, can we change the setting to disable it from the wdio-jasmine-framework?
Thanks.

**Comment by abdulnasar85** [comment]
Yes with 0.3.3 it gets executed in a random order. Going back to 0.3.2.

**Comment by Mechanicalkeyboard** [comment]
I was able to run jasmine init, and then configure the setting in my project.

**Comment by dudaming** [comment]
Hi,
I would like to help.
In Jasmine, you can disable this by setting the random: false in conf file. I didnot find the conf file in the wdio-jasmine-framework. Could you please guide me how to fix it?
Thanks

**Comment by giridharangm** [comment]
@dudaming @christian-bromann I had a PR ready for this. But now only I saw @dudaming 's comment.  

@dudaming, if you still wish to open a PR:  

open this file:  
> lib/adapter.js  

right after we initialize `Jasmine()`, add the following:  
``` javascript  
const { random = true } = this.jasmineNodeOpts // Optional, but recommended. To have jasmine's default behavior. 
this.jrunner.randomizeTests(this.jasmineNodeOpts.random) 
```  
That's it!.  

Now you can control the ordering of tests from `jasmineNodeOpts` in `wdio.conf.js`, by having a property called `random`.

**Comment by christian-bromann** [comment]
@giridharangm please open a PR for this. Thanks!

**Comment by giridharangm** [comment]
@christian-bromann done!  

 I've few suggestions on the tooling (devDependencies) that we're using right now. For example `babel-preset-es2015` / `stage-0` can be  dropped. Instead, we could use the `env` preset.  Also, it looks like we're running `rm -rf` in package scripts which will cause conflicts in Windows. We can use `rimraf` instead.  

Do you want to include those in the same PR or a new one?
